islandora_basic_collection.module

```diff
@@ -41,7 +41,9 @@ function islandora_basic_collection_share_migrate_access(AbstractObject $object)
   );
 
   if ($is_a_collection) {
-    return FALSE;
+    if ($object->id == "islandora:root") {
```

Comment:
I... know [this is how it is upstream](https://github.com/Islandora/islandora_solution_pack_collection/pull/209/files#diff-f7990cf10d52b3d4d46d8c0e9daaafa2R148); however... looking at it now, looks almost like it should reference our configured [`islandora_repository_pid`](https://github.com/discoverygarden/islandora/blob/31bcabc62c51f2d62d9761b0704f1583c1070735/config/install/islandora.settings.yml#L9)... Shrug?

... Probably fine here, since strictly, this is a port, just... weird to see...